libs/clients/work-machines/src/lib/providers.ts

```diff
@@ -1,14 +1,93 @@
+import {
+  XRoadConfig,
+  IdsClientConfig,
+  LazyDuringDevScope,
+} from '@island.is/nest/config'
+import { ConfigType } from '@nestjs/config'
 import {
   Configuration,
+  MachineCategoryApi,
+  MachineOwnerChangeApi,
+  MachineSupervisorChangeApi,
   MachinesApi,
   MachinesDocumentApi,
 } from '../../gen/fetch'
-import { ApiConfig } from './api.config'
+import { WorkMachinesClientConfig } from './workMachines.config'
+import { createEnhancedFetch } from '@island.is/clients/middlewares'
+
+const ConfigFactory = (
```

Comment:
Looks like a copy-paste from `api.config.ts`

Reply:
Yes, I did move config from the api.config.ts to this file.
Will remove this file => api.config.ts